modules/apps/portal-security-sso/portal-security-sso-openid-connect-test/src/testIntegration/java/com/liferay/portal/security/sso/openid/connect/internal/test/OIDCUserInfoProcessorTest.java

```diff
@@ -127,53 +142,86 @@ public void setUp() throws Exception {
 		_uuid = PortalUUIDUtil.generate();
 	}
 
+	@FeatureFlag("LPD-20879")
 	@Test
 	public void testProcessUserInfo() throws Exception {
+		String pid = ConfigurationTestUtil.createFactoryConfiguration(
+			"com.liferay.portal.security.sso.openid.connect.internal." +
+				"configuration.OpenIdConnectProviderConfiguration",
+			HashMapDictionaryBuilder.<String, Object>put(
+				"companyId", _companyId
+			).put(
+				"discoveryEndpoint", _DISCOVERY_ENDPOINT
+			).put(
+				"fallbackMatcher", "screenName"
+			).put(
+				"openIdConnectClientId", _clientId
+			).build());
+
+		String originalPropertyValue = PrefsPropsUtil.getString(
+			_companyId, PropsKeys.USERS_EMAIL_ADDRESS_REQUIRED);
```

Comment:
Perhaps you can use this `PrefsPropsTestUtil.swapWithSafeCloseable(...)` instead.